sr_robot_commander/src/sr_robot_commander/grasp_saver_unsafe.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+from sys import argv
+
+import rospy
+from moveit_msgs.srv import SaveRobotStateToWarehouse as SaveState
+from sensor_msgs.msg import JointState
+from moveit_msgs.msg import RobotState
+from sr_robot_commander.sr_arm_commander import SrArmCommander
+from sr_robot_commander.sr_hand_commander import SrHandCommander
+from sr_utilities.hand_finder import HandFinder
+
+"""
+***DISCLAIMER***
+
+This script provides a command line interface to save the current pose of the robot (made of an arm and hand).
```

Comment:
Add this blurb to a README and link from main [README](https://github.com/shadow-robot/sr_interface/blob/indigo-devel/README.md) to point to this functionality